index.html

```diff
@@ -124,5 +124,32 @@
     <p class="navbar-cta navbar-item right" id="navbar-cta-text">We're hiring</p>
   </nav>
 
+  <div id="hiring-section">
+    <div id="join-hiring">
+      <h3> We're Hiring </h3>
+      <p> Help us redefine the calendar, reclaim our schedules, and make the time we spend together, better </p>
+      <form action="#">
+        <button type="submit" id="roles-button">See the roles</button>
+      </form>
+    </div>
+
+    <div id="picture-join">
+      <img id="flying-shoe" src="images/joinbox-flying-shoe.png" alt="flying shoe">
+      <img id="pencil-small" src="images/joinbox-pencil-small.png" alt="pencil small">
+      <img id="green-box" src="images/joinbox-abstract-green-box.png" alt="abstract green box">
+      <img id="blue-ball" src="images/joinbox-abstract-blue-ball.png" alt="abstract blue ball">
+      <img id="flying-febby" src="images/joinbox-flying-febby.png" alt="flying febby">
+    </div>
```

Comment:
Think about how you'll manage in the future for a while. All the images in the website are *over* the background. 

If you use only id's for every image and id's for every section containing images, you'll need to repeat styles between all of those.

Now think about what happens if there's a bug or something, and you have to change id's and the css in a case by case basis, one at a time. 

Think about how much time you could've saved by using a class or another selector which applies to all of those images. You can just go a change a single piece of code if anything were to get buggy.

Classes are for things with similar properties and for saving trouble. Use them. Don't just throw id's at elements.